Sources/BeaconChain/Constants.swift

```diff
@@ -1,46 +1,54 @@
 import Foundation
 
+// Misc
 let SHARD_COUNT = UInt64(2**10)
 let TARGET_COMMITTEE_SIZE = UInt64(2**7)
-let EJECTION_BALANCE = UInt64((2**4) * Int(1e9))
+let EJECTION_BALANCE = UInt64(2**4 * UInt64(1e9))
```

Comment:
In Swift the naming convention for constants is the same as variables. Maybe just encapsulate in `struct Constants{...}`. They should not be all uppercase.

Reply:
This file will be changed to a configuration file later on.